docs/content/concepts/assets/external-assets.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: External Assets | Dagster
+description: External assets model assets in Dagster that are not scheduled or materialized in Dagster.
+---
+
+# External assets (Experimental)
+
+An external asset is an asset that is not materialized by Dagster, but is tracked in the asset graph and asset catalog. This allows you to model assets in Dagster, attach metadata and events to those assets, but without scheduling their materialization with Dagster.
+
+Example use cases:
+
+- Data that is landed by an external source (e.g. a external file landing daily; Kafka landing data into s3)
+- Data that is created and processed using manual processes
+- Data that is materialized by existing pipelines with their own scheduling and infrastructure that you do not want to or need to migrate en masse.
+
+With an external assets you can:
+
+- Attach metadata to its definition for documentation, tracking ownership, and so on
+- Track its data quality in Dagster.
+- Track its data version in Dagster
+- Schedule downstream assets based on updates to external assets using asset sensors or auto-materialize policies
+
+But you cannot
+
+- Schedule its materialiation
+- Backfill it using Dagster
+- Use the UI or GraphQL API to instigate ad hoc materializations.
```

Comment:
```suggestion
You cannot, however:

- Schedule an external asset's materialiation
- Backfill an external asset using Dagster
- Use the Dagster UI or GraphQL API to instigate ad hoc materializations.
```

Suggesting a few tweaks here. I found reading the intro a bit awkward due to how the info is chunked + worded, but this may help with flow.